packages/google_maps_flutter/lib/src/controller.dart

```diff
@@ -173,6 +173,20 @@ class GoogleMapController {
     });
   }
 
+  /// Sets the styling of the base map.
+  ///
+  /// Set to `null` to clear any previous custom styling.
+  ///
+  /// Returns `true` if the style was successfully parsed; `false` if problems were
+  /// detected with the [mapStyle], including un-parsable styling JSON, unrecognized
+  /// feature type, unrecognized element type, or invalid styler keys. If the return
+  /// value is `false`, the current style is left unchanged.
+  ///
+  /// The style string can be generated using https://mapstyle.withgoogle.com/
```

Comment:
Is there any documentation for the json format we can refer to?

Reply:
Here's the iOS style reference: https://developers.google.com/maps/documentation/ios-sdk/style-reference
I believe it is identical to Android: https://developers.google.com/maps/documentation/android-sdk/style-reference

Reply:
Thanks @domesticmouse , will add these to the docs.